easybuild/easyconfigs/n/NGS/NGS-2.10.9-GCC-10.2.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
+
+name = 'NGS'
+version = '2.10.9'
+
+homepage = 'https://github.com/ncbi/ngs'
+description = """NGS is a new, domain-specific API for accessing reads, alignments and pileups
+produced from Next Generation Sequencing."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
```

Comment:
Do we want to move this into GCC instead of GCCcore?

Reply:
No, sorry.  Moving back to GCCcore.